packages/serverpod/lib/src/database/adapters/postgres/database_connection.dart

```diff
@@ -665,9 +671,30 @@ Current type was $T''');
   return table!;
 }
 
+_PostgresTransaction? _getPostgresTransactionOrAssert(
```

Comment:
Suggestion: rename this method it neither gets a transaction or asserts it, validateTransaction I think is more descriptive here.